README.md

```diff
@@ -150,6 +150,10 @@ other technologies change the origin of a request that a web server can see.
   This strategy simply looks at the IP of a request and uses that to extract
   more data using IPinfo.
 
+- `XForwardedFor`
```

Comment:
We should have a new section for this the way we have for other strategies, and it should be named accordingly, like `XForwardedForIPStrategy`.

Reply:
Oh right.